internal/pkg/plugin/jenkinspipelinekubernetes/validate.go

```diff
@@ -24,22 +40,24 @@ func validate(options *Options) []error {
 
 func defaults(options *Options) {
 	// pre handle the options
-	if options.J.PipelineScriptPath == "" {
-		options.J.PipelineScriptPath = defaultJenkinsPipelineScriptPath
+	if options.JenkinsfilePath == "" {
+		options.JenkinsfilePath = defaultJenkinsPipelineScriptPath
 	}
 
-	if options.J.User == "" {
-		options.J.User = defaultJenkinsUser
+	if options.JenkinsUser == "" {
+		options.JenkinsUser = defaultJenkinsUser
 	}
 
-	options.J.URL = "http://" + options.J.URL
+	if !strings.Contains(options.JenkinsURL, "http") {
+		options.JenkinsURL = "http://" + options.JenkinsURL
+	}
```

Comment:
`url` tag do not need this default value set. It must have prefix `http(s)://`

Reply:
the "defaults" logic is before the "validate".

Reply:
![image](https://user-images.githubusercontent.com/18692628/186796607-b00be6cd-7448-43b6-9b9a-1dbf04642b70.png)